lib/modules/list-widgets/views/widget.html

```diff
@@ -2,15 +2,27 @@
   {{data.widget.formattedContainerStyles}}
 </style>
 
-<ul id="{{data.widget.containerId}}" class="{{data.widget.listClassName}}">
+{% if data.widget.listClassName === 'numbers' %}
```

Comment:
Deze kan nooit 'numbers' zijn, als je namelijk de Numbers list type selecteert is de waarde 'uk-list uk-list-numbers'